11-Backbonejs/03-Backbone-Models.md

```diff
@@ -67,44 +68,52 @@ console.log("The person is named " + ada.get("name"));
 ```
 
 Results in:
-
 ```console
 Person has been Created
 The person is named Ada
 ```
 
-You can also pass in attributes to a model at instantiation via a JSON object in the parameters to new.
+You can also pass in **attributes** to a model at instantiation via a JSON object in the parameters to new.
 
 ```javascript
 var babbage = new Person({name: "Charles Babbage", age: 65, skills: "Hardware Design, Mathematics, Flower Arrangement."});
 ```
 
+These attributes do not necessarily need to have default values specified, or really be specified in any way. In the example above, we've provided two of the fields that we decided to set defaults for, and added another, `skills`.
+
 ## Get, Set & Unset
 
-Once you have an instance of your Model you can use get & set methods to set it's attributes.
+Once you have an instance of your Model you can use `get` & `set` methods to set it's attributes. You may pass individual keys with values, or you may pass an object containing any and all data that you might want to change.
 
+Setting **each key/value**:
 ```javascript
 ada.set("skills", "Programming, Mathematics, Mountain Climbing.");
 ada.set("phone", "(444) 465-9122");
+```
 
-// prints '(444) 465 9122'
-console.log(ada.get("phone"));
-// prints 'Programming, Mathematics, Mountain Climbing.'
-console.log(ada.get("skills"));
+Setting using an **object**:
+```javascript
+ada.set({
+  skills: "Programming, Mathematics, Mountain Climbing.",
+  phone: "(444) 465-9122"
+});
+```
 
+```javascript
+console.log(ada.get("phone")); // '(444) 465 9122'
+console.log(ada.get("skills")); // 'Programming, Mathematics, Mountain Climbing.'
 ```
 
 You can use `unset` to remove an attribute.
 ```javascript
 ada.unset("skills");
 
-// prints undefined
-console.log(ada.get("skills"));
+console.log(ada.get("skills")); // undefined
 ```
 
 ## Adding Additional Methods to a Model
 
-You can add additional models to a method like other attributes:
+You can add additional methods to a Model like other attributes:
```

Comment:
We should probably say functions instead of methods, and properties instead of attributes.